src/Zicht/Bundle/AdminBundle/Controller/CRUDController.php

```diff
@@ -48,7 +48,10 @@ public function duplicateAction()
         $objectManager->persist($newObject);
         $objectManager->flush();
 
-        $this->addFlash('sonata_flash_success', 'flash_duplicate_success');
+        $this->addFlash(
+            'sonata_flash_success',
+            $this->admin->trans('flash_duplicate_success', [], 'admin')
```

Comment:
Zou handig zijn om een default message toe te voegen in `admin.{nl|en}.yml`

Reply:
Ik zou admin domain ook weg laten, dit geeft wat meer flexibiliteit om dat te laten bepalen in de onderliggende admin .

Dit is ook wat we ook eigenlijk altijd doen in de service definitie door de method call `setTranslationDomain` op admin te zetten. Wat gebruikt zal worden als je de 3e parameter weglaat.

Reply:
Maar dan zou niet werken zonder dat je expliciet een `setTranslationDomain` call configureert in je project? Of kan het ook zonder?

Reply:
Default staat hij op `messages`, zie:

3.x => https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Admin/AbstractAdmin.php#L150
2.x => https://github.com/sonata-project/SonataAdminBundle/blob/2.x/Admin/Admin.php#L188

Reply:
Ok, dus er gaat niks fout in projecten als we `'admin'` weglaten? Dan kunnen we dat nog even aanpassen. En in het betreffende project `setTranslationDomain('admin')` configureren (of de vertaling in `messages.xx.yml` zetten :wink:).